.devops/deploy-pipelines.yml

```diff
@@ -0,0 +1,151 @@
+# Deploy to Azure Kubernetes Service
+# Build and push image to Azure Container Registry; Deploy to Azure Kubernetes Service
+# https://docs.microsoft.com/azure/devops/pipelines/languages/docker
+
+trigger:
+  branches:
+    include:
+      - master
+      - develop
+      - release-*
+      - features/*
+      - hotfix/*
+  paths:
+    include:
+      - api/*
+      - app/*
+      - core/*
+      - integration/*
+      - pom.xml
+
+pr: none
+
+resources:
+  - repo: self
+
+variables:
+  # dockerfileRelativePath: '$(DOCKERFILE)' # DockerfileV1
```

Comment:
If its not needed as it seems please remove this line

Reply:
OK, thank you. I'll fix